ssh_agent_unix.go

```
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"net"
+
+	"golang.org/x/crypto/ssh/agent"
+)
+
+func dialSSHAgent(authSock string) (agent.ExtendedAgent, io.Closer, error) {
+	conn, err := net.Dial("unix", authSock)
```

golangci-lint (2.12.1)

[error] net.Dial must not be called. use (*net.Dialer).DialContext

(noctx)